setup.py

```diff
@@ -21,6 +21,7 @@
         'pandas',
         'numpy',
         'requests',
-        'iso8601'
+        'iso8601',
+        'setuptools'
```

Comment:
unfortunately for the check I need `pkg_resources` which is part of `setuptools` - so I should include it as a install dependancy